front/src/applications/editor/Editor.tsx

```diff
@@ -441,6 +441,7 @@ const Editor = () => {
                   activeTool: toolAndState.tool,
                 }}
                 viewPort={viewport}
+                compact={false}
```

Comment:
In MapButtons, set compact to false by default and remove this line (and in referenceMap too)
```suggestion
```